applications/band-grants.md

```diff
@@ -0,0 +1,167 @@
+
+### Team Code Repos
+
+- https://github.com/bandprotocol/
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+
+- https://github.com/warittornc
+- https://github.com/taobun
+
+### Team LinkedIn Profiles (if available)
+
+- Sireethorn: linkedin.com/in/sireethorn-fon-satchatippavarn-11792aaa
+- Buranapong: linkedin.com/in/buranapongburanasiri
+- Sirada: linkedin.com/in/sirada-lorhpipat
+- Warittorn: linkedin.com/in/warittorn-cheevachaipimol-467878230
+
+
+## Development Status :open_book:
+
+- links to your research diary, blog posts, articles, forum discussions or open GitHub issues,
+- references to conversations you might have had related to building this template,
+    - Band Partners: https://www.bandpartners.io/
+    - Blog: https://blog.bandprotocol.com/
+    - Github: https://github.com/bandprotocol/
+
+## Development Roadmap :nut_and_bolt:
+
+This section should break the development roadmap down into milestones and deliverables. Since these will be part of the agreement, it helps to describe _the functionality we should expect in as much detail as possible_, plus how we can verify and test that functionality. Whenever milestones are delivered, we refer to this document to ensure that everything has been delivered as expected.
+
+Below we provide an **example roadmap**.
+
+### Overview
+
+- **Total Estimated Duration:** 3-4 months
+- **Full-Time Equivalent (FTE):**  2
+- **Total Costs:** 80,000 USD
+
+### Milestone 1
+- **Estimated duration:** 2-3 months
+- **FTE:**  2
+- **Costs:** 45,000 USD
+
+|  Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License | MIT 
+| **0b.** | Documentation | We will provide inline documentation of the code, a basic tutorial on how to use the smart contract and an example consumer contract that shows a practical example on how to use the smart contract|
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests.|
+| **0d.** | Docker | N/A for smart contracts|
+| 0e. | Article | We will publish an **article**/workshop that explains [...] (what was done/achieved as part of the grant). (Content, language and medium should reflect your target audience described above.) |
+| 1. | Data Source | Data sources on BandChain which will be the core of getting data that will be delivered to Astar |
+| 2. | Oracle Script | An oracle script on BandChain which will verify and aggregate data from the data sources on BandChain|
+| 3. | PDS | A service on BandChain that will allow paid services to deliver data to BandChain safely and securely |
+| 4. | Smart contract | An ink! smart contract that will allow consumers to query various price feed from BandChain |
+| 5. | Testing and Audit | Unit tests for each component and live testing and audits for the ink! smart contract on testnet |
```

Comment:
The audit wouldn't be part of this milestone.

You should have unit tests and some sort of automated integration test suite